panel/param.py

```diff
@@ -214,35 +213,41 @@ class Param(Pane):
         Dictionary of widget overrides, mapping from parameter name
         to widget class.""")
 
-    mapping: ClassVar[Mapping[param.Parameter, Widget | Callable[[param.Parameter], Widget]]] = {
-        param.Action:            Button,
-        param.Array:             ArrayInput,
-        param.Boolean:           Checkbox,
-        param.Bytes:             FileInput,
-        param.CalendarDate:      DatePicker,
-        param.Color:             ColorPicker,
-        param.Date:              DatetimeInput,
-        param.DateRange:         DatetimeRangeSlider,
-        param.CalendarDateRange: DateRangeSlider,
-        param.DataFrame:         DataFrameWidget,
-        param.Dict:              LiteralInputTyped,
-        param.FileSelector:      SingleFileSelector,
-        param.Filename:          TextInput,
-        param.Foldername:        TextInput,
-        param.Integer:           IntSlider,
-        param.List:              LiteralInputTyped,
-        param.MultiFileSelector: FileSelector,
-        param.ListSelector:      MultiSelect,
-        param.Number:            FloatSlider,
-        param.ObjectSelector:    Select,
-        param.Parameter:         LiteralInputTyped,
-        param.Range:             RangeSlider,
-        param.Selector:          Select,
-        param.String:            TextInput,
-    }
-
-    if hasattr(param, 'Event'):
-        mapping[param.Event] = Button
+    @classproperty
```

Comment:
Users could override the mappings in here previously.

Reply:
Should be possible again with: [8286744](https://github.com/holoviz/panel/pull/7207/commits/82867442a672c42352805b38cb0c667ad64066dd)

(The dict merge was added in Python 3.9)